engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/JobRestServiceInteractionTest.java

```diff
@@ -142,24 +147,45 @@ public void setUpRuntimeData() {
 
   @Test
   public void testSetJobRetries() {
-    Map<String, Object> retriesVariableJson = new HashMap<String, Object>();
+    Map<String, Object> retriesVariableJson = new HashMap<>();
+    retriesVariableJson.put("retries", MockProvider.EXAMPLE_JOB_RETRIES);
+
+    given().pathParam("id", MockProvider.EXAMPLE_JOB_ID).contentType(ContentType.JSON).body(retriesVariableJson).then().expect()
+    .statusCode(Status.NO_CONTENT.getStatusCode())
+    .when().put(JOB_RESOURCE_SET_RETRIES_URL);
```

Comment:
Wow, this is not a beauty 😄 Looks like this test class uses this style consistently here, unfortunately.
I still suggest we format this like we usually do on separate lines with proper indentation.
You can adjust the existing test methods as well if you like. For new test methods, let's use a format that is a bit more readable.